reference/test/factorization/lu_kernels.cpp

```diff
@@ -216,7 +218,7 @@ TYPED_TEST(Lu, KernelFactorizeWorks)
         gko::kernels::reference::lu_factorization::factorize(
             this->ref, this->storage_offsets.get_const_data(),
             this->row_descs.get_const_data(), this->storage.get_const_data(),
-            diag_idxs.get_const_data(), this->mtx_lu.get(), tmp);
+            diag_idxs.get_const_data(), this->mtx_lu.get(), false, tmp);
```

Comment:
Since this is an `Lu` test, shouldn't the next-to-last parameter be `true`? (`full_fillin` = `true`)?